### DISCUSSION TITLE
posts/2020/sphinx-blogging/

### DISCUSSION THREAD

**Discussion by giscus** [choldgraf/choldgraf.github.io#30] [Blog comments]
# posts/2020/sphinx-blogging/

I recently re-wrote all of the infrastructure for my blog so that it now builds on top of the Sphinx ecosystem! This is a short post to describe the reasons for doing so, and a bit about the implem...

https://predictablynoisy.com/posts/2020/sphinx-blogging/

**Comment by grisaitis** [comment]
Hi! Thanks for sharing this about ablog. At the end you mention sharing a bit more about jupyter book as a part of this system, but I didn't see anything :) Any chance you have more to share? Do you recommend using jupyter book as part of this workflow?
